provider/ansible/resource_override.rb

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'provider/resource_override'
+
+module Provider
+  module Ansible
+    class ResourceOverride < Provider::ResourceOverride
```

Comment:
Did you run Rubocop? Documentation for this class is missing: `rake test:rubocop` and please make sure it is error+warning free.

Suggestion: Instead of adding the TODO below, add documentation explaining what the class does and leave the class empty.

Reply:
Yes I did. There is 21 offenses (@rambleraptor you should fix them ;) ), but none for this file.

I will add documentation before I submit this PR.